ember/ember-cli-build.js

```diff
@@ -10,6 +10,10 @@ module.exports = function(defaults) {
       extensions: ['css', 'js'],
       exclude: ['cesium'],
     },
+    minifyJS: {
+      enabled: EmberApp.env() !== 'production',
```

Comment:
hmm, I just noticed, shouldn't the check be `EmberApp.env() === 'production'` instead of `!==`?